exchange/exchange.go

```diff
@@ -350,12 +350,15 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 		r.HookExecutor.ExecuteAllProcessedBidResponsesStage(adapterBids)
 
 		if targData != nil {
+			multiBid, multiBidErrs := getExtMultiBidData(requestExt)
```

Comment:
This function creates a map of multi-bids and while doing so, we're validating the multibid config passed in the request. IMO, any kind of validation should be done at the request processing layer rather than the response processing layer. I think a better place to add this validation might be in `ValidateRequestExt` function in `openrtb2/auction.go`

Reply:
Yeah I think it makes sense to have this validation be done at the request layer

Reply:
Done, moved the validation of incoming multibid ext to `ValidateRequestExt` and updated the downstream multibid ext to avoid rework.